packages/extension-ui/src/Popup/Forget.tsx

```diff
@@ -40,10 +40,7 @@ function Forget ({ className, match: { params: { address } } }: Props): React.Re
       />
       <div className={className}>
         <Address address={address}>
-          <Warning
-            className='movedWarning'
-            isDanger
```

Comment:
Same here, this is a warning, not an error, removing the isDanger. The Button is still red.